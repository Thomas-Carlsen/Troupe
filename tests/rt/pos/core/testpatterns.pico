let
    val proc1 = fn () => printWithLabels 22
    val _ = printWithLabels 32
    val pid2 = spawn(proc1)
in 0
end
