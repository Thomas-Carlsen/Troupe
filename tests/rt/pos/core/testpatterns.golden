2019-03-07T12:08:12.109Z [RTM] [32minfo[39m: Skipping network creation. Observe that all external IO operations will yield a runtime error.
32@{}%{}
>>> Main thread finished with value: 0@{}%{}
22@{}%{}
